cmd/troubleshoot/dynakube_test.go

```diff
@@ -259,28 +261,28 @@ func (builder *testDynaKubeBuilder) withCustomPullSecret(secretName string) *tes
 }
 
 func (builder *testDynaKubeBuilder) withProxy(proxyURL string) *testDynaKubeBuilder {
-	builder.dynakube.Spec.Proxy = &dynakube.DynaKubeProxy{
+	builder.dynakube.Spec.Proxy = &common.ValueSource{
```

Comment:
here for example: common.ValueSource and I prefet to name it: `proxy` or something like it, prev it was clear 
Proxy = dynakube.DynakubeProxy

Reply:
the struct itself has 0 relation to proxy :D 

`ValueSource` is the "old" `DynaKubeProxy` and `DynaKubeValueSource`, as these where the exact same

there is the `ProxySpec` that is 100% proxy related, and is used in EdgeConnect, and hopefully will be used in the DynaKube in the future

Reply:
`DynaKubeProxy` only exists for "historical reasons", which I would prefer not to keep. 😅

Reply:
okey, you can add a comment if you want. but we can resolve it.